src/Consumer/Process.php

```diff
@@ -137,7 +135,9 @@ public function start()
      */
     public function stop()
     {
-        $this->isRunning = false;
+        // TODO: we should remove the consumer from the group here
```

Comment:
@nmred I need a bit of help here to understand how we can plug the `LeaveGroup` message in here

Reply:
@lcobucci  you can refer #120, We can discuss this